src/locales.json

```diff
@@ -420,13 +424,17 @@
     "en": {
       "troubleshooting": "Help to improve Integreat",
       "troubleshootingDescription": "Automatically send crash reports",
+      "allowPushNotifications": "Get Notifications",
+      "allowPushNotificationsDescription": "Would you like to get notifications for the latest local updates and news?",
```

Comment:
I think these message should be the same in the dialog and the settings for now. The should have different keys but the same content.